aws/v11.5.5/release.diff

```diff
@@ -0,0 +1,66 @@
+  - componentVersion: 3.5.0                                          - componentVersion: 3.5.0
+    name: kiam                                                         name: kiam
+    version: 1.3.0                                                     version: 1.3.0
+  - componentVersion: 1.9.5                                          - componentVersion: 1.9.5
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.1.0                                                     version: 1.1.0
+  - componentVersion: 0.3.3                                          - componentVersion: 0.3.3
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.0                                                     version: 1.1.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 0.18.1                                         - componentVersion: 0.18.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.2.0                                                     version: 1.2.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: aws-cni                                                    - name: aws-cni
+    version: 1.6.0                                              |      version: 1.7.3
+  - name: aws-operator                                               - name: aws-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 8.7.6                                              |      version: 8.7.7
+  - name: calico                                                     - name: calico
+    version: 3.15.1                                                    version: 3.15.1
+  - name: cert-operator                                              - name: cert-operator
+    reference: 0.1.0-2                                                 reference: 0.1.0-2
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.1.0                                                     version: 0.1.0
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 2.3.4                                                     version: 2.3.4
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.2.1                                                  version: 2512.2.1
```

Comment:
The containerlinux version is still the same. Is this normal? Shouldn't it be '2512.5.0' which is the latest stable version?